source/index.lagda

```diff
@@ -111,6 +111,8 @@ import UF.index
 import Uncatalogued.index
 import Utilities.index
 
+import Modal.index                    -- by Jon Sterling
```

Comment:
I'd rather have the imports in alphabetical order. Thanks.

Reply:
oops! Done 😄

Reply:
Thanks.